integration/kubernetes/operator/presto/config/manager/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+resources:
+- manager.yaml
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+- files:
+  - controller_manager_config.yaml
+  name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: beinan6666/presto-operator
```

Comment:
operator-sdk would help to build and push this image to the specified repository after executing the following command. So we need to make sure this repository is available in the production environment. Otherwise, it would fail to deploy this operator in the Kubernetes cluster. 
```shell
make install && make deploy
```